knowledge-base/customize-selected-tab-border-tabstrip-blazor.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Customizing the Border of the Selected Tab in TabStrip
+description: Learn how to modify the border appearance around the selected tab and its content in the TabStrip component for Blazor.
+type: how-to
+page_title: How to Change the Border of the Selected Tab in Blazor TabStrip
+slug: customize-selected-tab-border-tabstrip-blazor
+tags: tabstrip, border, customization, blazor, tab
+res_type: kb
+ticketid: 1652526, 1563024, 1574295, 1620296, 1647194
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>TabStrip for Blazor</td>
+	    </tr>
+    </tbody>
+</table>
+
+## Description
+
+When using the [TabStrip](https://docs.telerik.com/blazor-ui/components/tabstrip/overview) for Blazor, the default appearance includes a grey outline around the selected tab. The requirement is to modify this outline to encompass both the tab title and the content of the selected tab, rather than just the tab title. 
```

Comment:
```suggestion
When using the [TabStrip](https://docs.telerik.com/blazor-ui/components/tabstrip/overview) for Blazor, the default appearance includes a grey outline around the selected tab. How can I modify this outline to encompass both the tab title and the content of the selected tab, rather than just the tab title. 
```